injective/assetlist.json

```diff
@@ -224,16 +224,15 @@
       "description": "GLTO-ERC20 on injective",
       "denom_units": [
         {
-          "denom": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+          "denom": "glto",
```

Comment:
We can't have 2 demons with difference decimal exponent, otherwise interfaces won't know how to display the asset. What is the name of the base (fundamental, indivisible) denomination of glto when it's on Injective? As in, what would it's CLI require?

Reply:
This is the base denom on INJ: https://explorer.injective.network/asset/?denom=peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2&tokenType=erc20

I updated the PR, since GLTO is 6 decimals, for the base denom should I leave it at 0 or 6?